jetty-setuid/jetty-setuid-jna/src/main/java/org/eclipse/jetty/setuid/Passwd.java

```diff
@@ -26,86 +28,49 @@
  *    };
  *
  */
-
-public class Passwd
+@Structure.FieldOrder({"_pwName", "_pwPasswd", "_pwUid", "_pwGid", "_pwGecos", "_pwDir", "_pwShell"})
```

Comment:
Would it be worth dropping the `_` prefix now?
Does it make things easier? or more complicated?

Reply:
The field names don't matter to JNA. Only these things matter when you map a C `struct` to a java `class`:
 - the fields' type
 - the fields' declared order in `@Structure.FieldOrder`
 - all fields _must_ be public
 - the class must extend from `Structure`

Since this class is not considered publicly-visible API, we can refactor it to your liking, even dropping the getters/setters altogether.